RuntimeDoom.cs

```diff
@@ -0,0 +1,130 @@
+using System;
+using System.Linq;
+using System.Collections;
+using System.Collections.Generic;
+using ManagedDoom;
+using ManagedDoom.Unity;
+using Xabblll.DoomInInspector;
+using UnityEngine;
+using UnityEngine.UI;
+using Sprite = UnityEngine.Sprite;
+
+public class RuntimeDoom : MonoBehaviour
+{
+    [SerializeField]
+    private int fps = 30;
```

Comment:
Doom engine operates at 35FPS. Better use it as default value

Reply:
So if i change it, it will be accepted?

Reply:
nvm, it already is accepted, thanks